### PR TITLE
TIG-2564 CrudActor to use one session per actor per phase

### PR DESCRIPTION
**Pull request by dalyd**

### CODE REVIEW

File: src/workloads/execution/CreateBigIndex.yml [hunk 1]
```diff
@@ -90,4 +90,7 @@ Actors:
 
 AutoRun:
   Requires:
-    mongodb_setup: [single-replica]
+    mongodb_setup:
+    - standalone
```
Comment (dalyd) [inline src/workloads/execution/CreateBigIndex.yml:94]: THis is a separate ticket and already pushed. I'm not sure why it's showing up here. Same with next ticket. I think it was just stupid on my local setup.

**Comment by dalyd** [comment]
I added some docs. Take a look and let me know what you think.

**Comment by rtimmons** [comment]
(Still) LGTM, @dalyd. Thanks for adding some docs. Please `evergreen merge` whenever you're ready.

**Comment by dalyd** [comment]
evergreen merge

**Pull request merged**